app/views/personal_details/create.html.erb

```diff
@@ -0,0 +1,20 @@
+<span class="govuk-caption-xl"><%= t('application_form.personal_details_section.heading') %></span>
+<h1 class="govuk-heading-xl"><%= t('application_form.review_answers') %></h1>
+
```

Comment:
```suggestion
```

nit: extra line break?

Reply:
This is now fixed and therefore outdated, but there's no outdated label, presumably because there's still a newline left to remove